posts/inside-rust/2022-05-19-governance-update.md

```diff
@@ -0,0 +1,95 @@
+---
+layout: post
+title: "Governance Update"
+author: Ryan Levick and Mara Bos
+---
+
+Last month, the core team, all the leads of top-level teams, the moderators, and the project representatives on the Rust Foundation board jointly sent out an update to all Rust project members on investigations happening into improvements to the governance of the Rust project. 
+
+As is detailed in the update, this report follows extensive conversations with many inside of the project and will continue as we encourage others in the project to reach out should they have feedback or want to get involved. 
+
+Since proper governance of the Rust project impacts all users of Rust, we thought it appropriate to also make this summary public which we have done below in an unaltered form. 
```

Comment:
```suggestion
Since proper governance of the Rust project impacts all users of Rust, we thought it appropriate to also make this summary public. While typos and grammatical fixes have been made since the original post, the text is otherwise unaltered; see https://github.com/rust-lang/blog.rust-lang.org/pull/974 for history starting from the text of the original email.
```

Reply:
(Might be better as a footnote:

```
… public.[^1]

[^1]: While typos and …
```
)

Reply:
Added a footnote.